lib/ui/Meeting/meet.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'dart:async';
+import 'package:flutter/services.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:iiit_sonepat_stable/constants/StringConstt.dart';
+import 'package:iiit_sonepat_stable/constants/routes.dart';
+import 'package:iiit_sonepat_stable/constants/validators.dart';
+
+class Meeting extends StatefulWidget {
+  _Meeting createState() => _Meeting();
+}
+
+class _Meeting extends State<Meeting> {
+  // Create a text controller and use it to retrieve the current value
+  // of the TextField.
+  final myController = TextEditingController();
+  static const platform = const MethodChannel('samples.flutter.dev/meet');
+
+  Future<void> _getLevel(text) async {
```

Comment:
Hey @Chandra-Mauli-Sharma, looks like we are not handling error here for the API call can we add some error handling here?

Reply:
`String _meetingStatus = 'Unknown meeting Status';

  Future<void> _getLevel(text) async {
    String meetingStatus;
    try {
     final int status=await  platform.invokeMethod('Meeting started', {"text": text});
      meetingStatus = 'Meeting Status $status % .';
    } on PlatformException catch (e) {
      meetingStatus = "Error in Joining Meeting '${e.message}'.";
    }
  }`

Reply:
@therealansh will this be fine

Reply:
Lgtm